src/commands.rs

```diff
@@ -13,13 +13,22 @@ pub struct Args {
 
 #[derive(Debug, Clone, Subcommand)]
 pub enum Subcommands {
+    Init(Init),
     Install(Install),
     Update(Update),
     Login(Login),
     Push(Push),
     VersionDryRun(VersionDryRun),
 }
 
+#[derive(Debug, Clone, Parser)]
+#[clap(
+    about = "Initialize a fresh Soldeer project.",
```

Comment:
probably we have to formalize how we refer to Soldeer, so probably we have to rename everything Soldeer